config/locales/fr.yml

```diff
@@ -39,6 +39,30 @@ fr:
     posts: Statuts
     remote_follow: Suivre à distance
     unfollow: Ne plus suivre
+  admin:
+    settings:
+      click_to_edit: Cliquez pour éditer
+      contact_information:
+        email: Entrez une adresse courriel publique
```

Comment:
This PR is solely about normalizing the order of locale keys. You can and should do it yourself by running `i18n-tasks normalize` after translating something so a PR like this wouldn't be needed.